starky/src/f5g.rs

```diff
@@ -29,15 +30,46 @@ impl F5G {
             dim: 5,
         }
     }
+}
+
+/// Number of bytes needed to represent field element
+const ELEMENT_BYTES: usize = core::mem::size_of::<u64>();
+
+impl FieldExtension for F5G {
+    const ELEMENT_BYTES: usize = ELEMENT_BYTES;
+    const IS_CANONICAL: bool = false;
+
+    const ZERO: Self = Self {
+        cube: [Fr::ZERO, Fr::ZERO, Fr::ZERO, Fr::ZERO, Fr::ZERO],
+        dim: 1,
+    };
+    const ONE: Self = Self {
+        cube: [Fr::ONE, Fr::ZERO, Fr::ZERO, Fr::ZERO, Fr::ZERO],
+        dim: 1,
+    };
```

Comment:
Please modify as comments in `F3G`.

Reply:
the definition of one() is not equal to the definition of ONE which has a difference in the property of dim( one().dim=5 and ONE.dim=1 )